README.md

```diff
@@ -1 +1,35 @@
-# TSH_FE
+# GH-Browser
+
+A browser for Github users. Entering github username will display user's details and history of his public events.
+
+## Installation
+
+Install npm packages:
+```
+yarn
+```
+
+
+## Develop locally
+
+1. Run:
+
+    ```
+    yarn start
+    ```
+
+2. In your browser, navigate to: [http://localhost:2000/](http://localhost:2000/)
+
+## Build production-ready app
+
+```
+yarn build
+```
+
+
+# Technical details
+
+* Bundler: [Webpack](https://webpack.js.org/)
+* DOM Manipulation: [Cash-DOM](https://github.com/kenwheeler/cash)
```

Comment:
We could have used more modern technologies like React, Typescript, but I do understand that this is business requirement :)